subprojects/parseq/src/main/java/com/linkedin/parseq/Task.java

```diff
@@ -1434,6 +1434,9 @@
    * will be resolved with results of all tasks as soon as all of them has
    * been completed successfully.
    *
+   * Tasks created using Task.blocking only run in parallel, tasks created using Task.callable are executed
+   * sequentially, although the visualization tools may show the task composition as parallel
```

Comment:
> although the visualization tools may show the task composition as parallel

This is actually not true. The video Amen forward(ed?) you should mention that the visualization tool actually tried to explain it is sequential, but the interval is so small that it looks like parallel.